modules/network-observability-updating-migrating.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-updating-migrating_{context}"]
+= Migrating removed stored versions of the CRD
+
+Before updating to the Network Observability Operator version 1.6, you need take steps to migrate the removed, stored version of the `FlowCollector` CRD. 
+
+.Procedure
+. Save the Network Observability Operator 1.5 version of the `FlowCollector` CR to a file, for example `flowcollector-1.5.yaml`. 
++
+[source,terminal]
+----
+$ oc get flowcollector cluster -o yaml > flowcollector-1.5.yaml
+----
+. Follow the steps in "Uninstalling the Network Observability Operator", which uninstalls the Operator and removes the existing `FlowCollector` CRD.
+. Install the Network Observability Operator latest version, 1.6.0.
+. Create the `FlowCollector` using backup that was saved in step 1 of this procedure. 
+
+.Verification
```

Comment:
@jotak I was unsure of the verification steps to take here. In a Slack thread, I saw you were thinking maybe `oc get crd [flowcollectors.flows.netobserv.io](http://flowcollectors.flows.netobserv.io/) -ojsonpath='{.status.storedVersions}'` but then crossed that out.

Reply:
exactly, cf my comment above

Reply:
Also, I would also mention the [Storage Version Migrator Operator](https://docs.openshift.com/container-platform/4.16/operators/operator-reference.html#cluster-kube-storage-version-migrator-operator_cluster-operators-ref) even if we don't provide detailed instructions, just so users know that solution exists

Reply:
Ok in addition to listing the diagnostic step in the other comment, I also added this as a Verification:

```
.Verification
* Run the following command:
[source,terminal]
----
$ oc get crd flowcollectors.flows.netobserv.io -ojsonpath='{.status.storedVersions}'
----
+
The list of results should no longer show `v1alpha1` and only show the latest version, `v1beta1`.
```